src/main/resources/org/jenkinsci/plugins/mesos/MesosSlaveInfo/ContainerInfo/config.jelly

```diff
@@ -1,10 +1,10 @@
-<?jelly escape-by-default='true'?>
+?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
     <f:entry title="${%Container Type}" field="type">
-        <f:radioBlock name="type" title="${%Docker}" value="DOCKER" inline="true" checked="true">
-            <f:entry title="${%Docker Image}" field="dockerImage" description="If using Docker, specify the docker image.">
+            <f:entry title="${%Docker Image}" field="dockerImage" description="Specify the docker image.">
```

Comment:
Is this change backwards compatible? If someone upgrades the plugin and used previous config, sounds like it would fail to start? I think we probably want to just add another entry for docker image name inside the new radioblock for compat.

Reply:
As discussed offline, we need to fix this for backwards compat.

Reply:
tested this and we should be good
- downgrades are not supported in jenkins so we are only concerned about uopgrade path
- When upgrading the containerType will be set to Docker correctly on the agent spec and the user can override this by selecting UCR